### ISSUE TITLE
Primary Object Type Code doesn't allow any values

### ISSUE THREAD

**Issue by daryllabar** [Innofactor/FetchXMLBuilder#160]
Using this xml:
```
<fetch top="50" >
  <entity name="sdkmessageprocessingstepimage" >
    <all-attributes/>
    <link-entity name="sdkmessageprocessingstep" from="sdkmessageprocessingstepid" to="sdkmessageprocessingstepid" >
      <link-entity name="sdkmessagefilter" from="sdkmessageid" to="sdkmessageid" >
        <filter>
          <condition attribute="primaryobjecttypecode" operator="eq" />
        </filter>
      </link-entity>
    </link-entity>
  </entity>
</fetch>
```

I have no way to specify the primaryobjecttypecode value.  When I run the actual query, it shows string values, as well as the early bound generation.  Maybe you have some hard-coded lookfup for that attribute to make it an optionset?

**Comment by rappen** [comment]
Not sure I follow what you are trying to do, and what is wrong/missing.
Could you elaborate?

**Comment by paulpleasant** [comment]
What he means is that FetchXMl Builder is interpreting primaryobjecttypecode as being a string and has no way to specify an entity. Just a big fat bug.

**Comment by rappen** [comment]
@paulpleasant ok thanks, can you verify the fix in version 1.2017.6.119 or later?
